CityOfDetroit/bloom
# feat: new verified tag icon

## Issue

- Closes #1124

## Description

A new icon for the tag on verified listings.

## How Can This Be Tested/Reviewed?

Create or view a verified listing (must not also be coming soon, since that tag takes precedence), and ensure you see the new check icon.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
